docs/source/azure/policy/resources/keyvault.rst

```diff
@@ -39,38 +39,12 @@ Actions
 Example Policies
 ----------------
 
-This set of policies will mark all Key Vaults for deletion in 7 days that have 'test' in name (ignore case),
-and then perform the delete operation on those ready for deletion.
+This policy will find all KeyVaults with 10 or less API Hits over the last 72 hours
```

Comment:
I think there are some nice examples around keys and permissions we can link here.

Reply:
There are other examples already! I just deleted the one that deletes in 7 days.